components/behavior/sticky/README.md

```diff
@@ -0,0 +1,44 @@
+# BehaviorSticky
+
+This component helps in the implementation of sticky components
```

Comment:
Could you add a link to the github of library we use on it? In order to have better documentation about how it works
https://github.com/garthenweb/react-stickup#readme